lib/jbuilder/jbuilder_template.rb

```diff
@@ -131,7 +135,7 @@ def set!(name, object = BLANK, *args)
     options = args.first
 
     if args.one? && _partial_options?(options)
-      _set_inline_partial name, object, options
+      _set_inline_partial name, object, options.dup
```

Comment:
Likewise here, but for `set!`. We can now safely mutate in `_set_inline_partial` to keep the other optimizations.